### PR TITLE
LPD-15573

### PR DESCRIPTION
**Pull request by brianikim**
Hello @smottal ,

https://liferay.atlassian.net/browse/LPD-15573, tied to LPP.

Straight-forward PR. We need to support GenerateWebDAVPasswordMVC Render/Action in UsersAdminPortlet.

Sincerely,
Brian I. Kim

### CODE REVIEW

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
Failures unrelated. all commerce tests passed.

File: modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts [hunk 1]
```diff
@@ -84,3 +84,20 @@ test('LPD-15224 check escape of memberships account name', async ({
...
+	editUserPage,
+	page,
+	usersAndOrganizationsPage,
+}) => {
+	await page.goto('/');
+
+	await usersAndOrganizationsPage.goToUsers();
+
+	await (await usersAndOrganizationsPage.usersTableRowLink('test')).click();
+
```
Comment (smottal) [inline modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts:98]: can we remove this empty line?

**Review by smottal — APPROVED** [review]
@brianikim just a very very little change if you have a chance to do it. Btw it's ok for me

**Comment by brianikim** [comment]
ci:test:sf

**Comment by brianikim** [comment]
ci:test:playwright

**Comment by brianikim** [comment]
ci:test:relevant

File: modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts (hunk 1 above)
Comment (brianikim) [reply modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts:97]: Done as requested

**Comment by brianikim** [comment]
Minor SF change completed as requested. Running CI

**Review by smottal — APPROVED** [review]
OK for me

**Comment by brianikim** [comment]
@smottal ,

Fairly certain the failures are unrelated for both playwright and CI.

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**